File: UI.MVC/Models/ConditionalPointDto.cs
using IP.Models.Dto;
using UI.MVC.Models.Dto;
using UI.MVC.Models.Dto.AnswerDto;

namespace UI.MVC.Models;

public class ConditionalPointDto
{
    public long ConditionalPointId { get; set; }
    public FlowStepDtoWassim Question { get; set; }
    public FlowStepDtoWassim FollowUpStep { get; set; }
    public AnswerDto Criteria { get; set; }

}

File: UI.MVC/ClientApp/src/ts/Configs/FlowStepConfig/conditionalpointdto.ts
import {FlowStepDto, QuestionDto} from "./flowstepDto";
import {AnswerDto} from "./AddAnswersInterfaces";

export interface ConditionalPointDto {
    Question: QuestionDto,
    Criteria: AnswerDto;
    FollowUpStep: FlowStepDto;
}
